spanner/spanner_snippets/spanner/spanner_query_information_schema_database_options.go

```diff
@@ -51,11 +53,11 @@ func queryInformationSchemaDatabaseOptions(w io.Writer, db string) error {
 		if err != nil {
 			return err
 		}
-		var OPTION_NAME, OPTION_VALUE string
-		if err := row.Columns(&OPTION_NAME, &OPTION_VALUE); err != nil {
+		var option_name, option_value string
+		if err := row.Columns(&option_name, &option_value); err != nil {
 			return err
 		}
-		fmt.Fprintf(w, "The result of the query to get %s for %s is %s", OPTION_NAME, databaseId, OPTION_VALUE)
+		fmt.Fprintf(w, "The result of the query to get %s for %s is %s", option_name, databaseId, option_value)
```

Comment:
nits: for Go's conventions, it should be `databaseID`. I guess you got this from the existing codebase. But better to fix it.